.github/workflows/lint-js-and-ruby.yml

```
@@ -45,12 +45,29 @@ jobs:
     - name: Install Node modules with Yarn for renderer package
       run: |
         yarn install --no-progress --no-emoji
+        sudo yarn global add yalc
+    - name: yalc publish for react-on-rails
+      run: yalc publish
+    - name: Save spec/dummy/node_modules to cache
+      uses: actions/cache@v4
+      with:
+        path: spec/dummy/node_modules
+        key: dummy-app-node-modules-cache-${{ hashFiles('spec/dummy/package.json') }}-${{ matrix.versions }}
```

actionlint (1.7.4)

property "versions" is not defined in object type {}

(expression)